data/adapters/eqtl_catalog_adapter.py

```diff
@@ -0,0 +1,157 @@
+                        name = 'modulates expression of'
+                        inverse_name = 'expression modulated by'
+                    else:
+                        label = 'splice_QTL'
+                        name = 'modulates splicing of'
+                        inverse_name = 'splicing modulated by'
+                    biological_context = f'ontology_terms/{row[4]}'
+                    studay = f'studies/{row[0]}'
+                    simple_sample_summaries = [row[5]]
+                    # example: ftp://ftp.ebi.ac.uk/pub/databases/spot/eQTL/susie/QTS000001/QTD000001/QTD000001.credible_sets.tsv.gz
+                    source_url = row[10]
+                    found_dataset = True
+                    break
+        if not found_dataset:
+            raise ValueError(f'No metadata found for dataset {dataset_id}')
+
+        with gzip.open(self.filepath, 'rt') as f:
+            self.writer.open()
+            qtl_reader = csv.reader(f, delimiter='\t')
+            next(qtl_reader)
+            for row in qtl_reader:
+                variant_vcf_format = row[3]
+                chr, pos, ref_seq, alt_seq = variant_vcf_format.split('_')
+                variant_id = build_variant_id(
+                    chr, pos, ref_seq, alt_seq, 'GRCh38'
+                )
+                gene_id = row[1]
+                is_valid_gene_id = self.gene_validator.validate(
+                    gene_id)
+                if not is_valid_gene_id:
+                    continue
+                # this edge id is too long, needs to be hashed
+                variants_genes_id = hashlib.sha256(
+                    (variant_id + '_' + gene_id + '_' + biological_context).encode()).hexdigest()
+                _props = {
+                    '_key': variants_genes_id,
+                    '_from': f'variants/{variant_id}',
+                    '_to': f'genes/{gene_id}',
+                    'biological_context': biological_context,
+                    'study': studay,
+                    'simple_sample_summaries': simple_sample_summaries,
+                    'label': label,
+                    'source': self.source,
+                    'source_url': source_url,
+                    'name': name,
+                    'inverse_name': inverse_name,
+                    'molecular_trait_id': row[0],
+                    'gene_id': gene_id,
+                    'credible_set_id': row[2],
+                    'variant_chromosome_position_ref_alt': variant_vcf_format,
+                    'rsid': row[4],
+                    'credible_set_size': int(row[5]),
+                    'posterior_inclusion_probability': float(row[6]),
+                    'pvalue': to_float(row[7]),
+                    'beta': to_float(row[8]),
+                    'standard_error': float(row[9]),
+                    'z_score': float(row[10]),
+                    'credible_set_min_r2': float(row[11]),
+                    'region': row[12]
```

Comment:
can also add intron_chr, intron_start, intron_end for splice QTLs (same as gtex_sqtl adapter, parse from molecular_trait_id here)